### DISCUSSION TITLE
[Idea] Add informations about docker file drive change.

### DISCUSSION THREAD

**Discussion by AiroPi** [nextcloud/all-in-one#4224] [Questions] [answered]
I changed recently the location of my docker data root to an external drive, and I faced an hard issue with collabora and capabilities.

I figured out that the solution was to remove the container (docker rm container) AND the image (docker image rm image) of collabora and then restart all the containers.

Because the solution was pretty hard to find, I think it would be interesting to add these informations somewhere. As you want.

**Comment by szaimen** [comment, marked as answer]
Hi 

> I think it would be interesting to add these informations somewhere. As you want.

Yes, maybe add them as notes to https://github.com/nextcloud/all-in-one?tab=readme-ov-file#how-to-store-the-filesinstallation-on-a-separate-drive ? Would you mind submitting a Pull Request? :)
